util/check_bin_md5sum

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+for tool in bin/*; do 
+	shasum=$(md5sum $tool)
+	util/update-modules $tool; 
+	rc="$?"
+	if [ $? -ne 0 ]; then
+		echo "'util/update-modules $tool' exited with code $rc"
+	fi
+	
+	updated_shasum=$(md5sum $tool)
+	if [ "$shasum" != "$updated_shasum" ]; then
+		echo "$tool had unsynced changes from lib, md5sum before and after running './util/update-modules $tool' is different"
+		exit 1
+	fi
```

Comment:
shouldn't we execute it for all tools and collect all possible failures?
So we can skip exit 1 and instead create an array of failed tools?

Reply:
We could yes, the idea was that if one tool was not updated, we should update-modules for all anyway

Would it help to have the full list ?

Reply:
Let's keep it as is